kale/prepdata/supergraph_construct.py

```diff
@@ -0,0 +1,250 @@
+import logging
```

Comment:
Some docstrings at the top of each module to summarize the scope will be better, e.g. https://github.com/pykale/pykale/blob/main/kale/prepdata/chem_transform.py

Reply:
added in [27b5b46](https://github.com/pykale/pykale/pull/338/commits/27b5b460b528ff6c675de7bb79669e850b387985).